doc/releases/changelog-dev.md

```diff
@@ -44,6 +44,9 @@
   [(#2709)](https://github.com/PennyLaneAI/pennylane/pull/2709)
 
 <h3>Improvements</h3>
+  
+* Added the possibility to group samples into counts providing `counts=True` flag in `qml.sample` function. This required creation of a new measurement type `Counts` in `measurements.py`.
+[(#2686)](https://github.com/PennyLaneAI/pennylane/pull/2686)
```

Comment:
Quick side comment, it might be useful to add a tiny example in the changelog as well. Just as an added way to advertise the feature outside of the doc-string. 

Something like the example you see for the `SPSAOptimizer` entry in the changelog, but not as big. Its not required, so feel free to pass. 

Cheers,

Reply:
Thanks a lot for the review!
I added the example from the docs into the changelog.